process.c

```diff
@@ -6839,19 +6844,21 @@ proc_setgroups(VALUE obj, VALUE ary)
 #ifdef HAVE_INITGROUPS
 /*
  *  call-seq:
- *     Process.initgroups(username, gid)   -> array
+ *     Process.initgroups(username, gid) -> array
  *
- *  Initializes the supplemental group access list by reading the
- *  system group database and using all groups of which the given user
- *  is a member. The group with the specified _gid_ is also added to
- *  the list. Returns the resulting Array of the GIDs of all the
- *  groups in the supplementary group access list. Not available on
- *  all platforms.
+ *  Sets the supplemental group access list;
+ *  the new list includes:
  *
- *     Process.groups   #=> [0, 1, 2, 3, 4, 6, 10, 11, 20, 26, 27]
- *     Process.initgroups( "mgranger", 30 )   #=> [30, 6, 10, 11]
- *     Process.groups   #=> [30, 6, 10, 11]
+ *  - The group IDs of those groups to which the user given by +username+ belongs.
+ *  - The group ID +gid+.
+ *
+ *  Example:
  *
+ *     Process.groups                # => [0, 1, 2, 3, 4, 6, 10, 11, 20, 26, 27]
+ *     Process.initgroups('me', 30 ) # => [30, 6, 10, 11]
```

Comment:
Remove space inside the parenthesis:

```suggestion
 *     Process.initgroups('me', 30)  # => [30, 6, 10, 11]
```